### ISSUE TITLE
Leap 16.0 Alpha: Can't install if btrfs snapshots are enabled (the default)

### ISSUE THREAD

**Issue by juliogonzalez** [agama-project/agama#1776]
As you can see at the screenshot:

![Screenshot_20241119_141158](https://github.com/user-attachments/assets/797e42f7-64ea-48a7-8d52-c34077a95f48)

As soon as I disable the snapshots, I can install the OS:

![Screenshot_20241119_141217](https://github.com/user-attachments/assets/9426d824-a2e7-405f-994d-bcbe63109c7d)

For the record, I initially tried installing from the ISO provided at https://get.opensuse.org/leap/16.0/#download, but I can reproduce it with the ISO from https://build.opensuse.org/projects/systemsmanagement:Agama:Devel/packages/agama-installer:openSUSE/repositories/images/binaries as well

**Comment by imobachgs** [comment]
Hi @juliogonzalez,

Thanks for reporting. This issue might be related to #1159.

**Comment by juliogonzalez** [comment]
> Hi @juliogonzalez,
> 
> Thanks for reporting. This issue might be related to #1159.

It looks to be, indeed, my systems have 20GB of RAM.

But I was scratching my head for a while until I just thought about disabling the snapshots... 

I am not changing we should change the defaults but... shouldn't we have a hint about the disk being too small for having btrfs snapshots?

**Comment by jreidinger** [comment]
I agree that we can add some hints how to require less space like removing snapshots, reducing swap size or removing separate home if it is proposed.

**Comment by mmaslano** [comment]
I can't even change btrfs to xfs or disable btrfs snapshot. Or delete / and create a new one. Is it expected?
![Screenshot_20241128_170733](https://github.com/user-attachments/assets/4293bf94-6fc6-49f7-be7a-6add42edce2a)

**Comment by kobliha** [comment]
@mmaslano You are using Agama 10 which might be simply too old. We are expecting Agama 11 to be out soon. Anyway, your issue is definitely something different than reported.

**Comment by ancorgs** [comment]
The new storage interface should make all this more clear. So closing for now.

Feel free to reopen if you disagree.

PS.- This could improve the situation even further, but it needs some love before merging it... and there are other priorities. https://github.com/ancorgs/agama/pull/2

**Comment by ancorgs** [comment]
Really closing now as for my previous comment.
